### DISCUSSION TITLE
cut flow table and impact

### DISCUSSION THREAD

**Discussion by jialin-guo1** [ram1123/2l2Q_limitSettingTool#60] [Comments_HIG_23_001]
| Zjet                  | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| Cut string name                   | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| nEvents                           | 7989994.00 | 9758.59 |
| passed_trig                       | 5668258.00 | -29.06 |
| passed_n_2ak4jets                 | 5668258.00 |   0.00 |
| passed_nmergedjets_lepclean       | 4306334.00 | -24.03 |
| passed_mergedjetsPt200            | 4306334.00 |   0.00 |
| passed_mergedjetEta2opint4        | 4304748.00 |  -0.04 |
| passed_n_2leps                    | 4260292.00 |  -1.03 |
| passed_flavor_charge              | 4226110.00 |  -0.80 |
| passed_lepZpt_40_25               | 3965488.00 |  -6.17 |
| passed_dR_lilj0point2             | 3965412.00 |  -0.00 |
| passed_Mll_4                      | 3965350.00 |  -0.00 |
| passed_lepisolation               | 3849274.00 |  -2.93 |
| passed_leptightID                 | 3830192.00 |  -0.50 |
| passed_lepclean                   | 3500990.00 |  -8.59 |
| passed_jetpt30                    | 3500990.00 |   0.00 |
| passed_lepZmass40_180             | 3416964.00 |  -2.40 |
| passed_lepZ                       | 3411694.00 |  -0.15 |
| passed_jeteta2opint4              | 3183394.00 |  -6.69 |
| passed_dijetpt100                 | 3182566.00 |  -0.03 |
| passed_mergedmass_40_180          | 1239278.00 | -61.06 |
| passed_ak4jetZm_40_180            | 1192508.00 |  -3.77 |
| passed_MergedolonlyHiggs          |  975182.00 | -18.22 |
| passed_particleNetZvsQCD0opint9   |   81046.00 | -91.69 |

| TTbar + tZq               | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| Cut string name                   | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| nEvents                           | 23544000.00 | 10650.59 |
| passed_trig                       | 15374668.00 | -34.70 |
| passed_n_2ak4jets                 | 15374668.00 |   0.00 |
| passed_lepclean                   | 10618944.00 | -30.93 |
| passed_jetpt30                    | 9544250.00 | -10.12 |
| passed_n_2leps                    | 8526718.00 | -10.66 |
| passed_flavor_charge              | 7981918.00 |  -6.39 |
| passed_jeteta2opint4              | 7970300.00 |  -0.15 |
| passed_lepZpt_40_25               | 5617062.00 | -29.53 |
| passed_dR_lilj0point2             | 5616978.00 |  -0.00 |
| passed_Mll_4                      | 5616804.00 |  -0.00 |
| passed_dijetpt100                 | 5466190.00 |  -2.68 |
| passed_lepisolation               | 5267034.00 |  -3.64 |
| passed_leptightID                 | 5177640.00 |  -1.70 |
| passed_ak4jetZm_40_180            | 4284624.00 | -17.25 |
| passed_nmergedjets_lepclean       | 1763770.00 | -58.83 |
| passed_mergedjetsPt200            | 1763770.00 |   0.00 |
| passed_mergedjetEta2opint4        | 1722696.00 |  -2.33 |
| passed_lepZmass40_180             | 1710600.00 |  -0.70 |
| passed_lepZ                       | 1697756.00 |  -0.75 |
| passed_mergedmass_40_180          |  771070.00 | -54.58 |
| passed_MergedolonlyHiggs          |  304400.00 | -60.52 |
| passed_particleNetZvsQCD0opint9   |  219002.00 | -28.05 |

| ZV+WW              | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| Cut string name                   | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| nEvents                           | 19988000.00 | 762800.76 |
| passed_trig                       | 10859842.00 | -45.67 |
| passed_n_2ak4jets                 | 10859842.00 |   0.00 |
| passed_n_2leps                    | 4925638.00 | -54.64 |
| passed_flavor_charge              | 4892272.00 |  -0.68 |
| passed_lepZpt_40_25               | 2574642.00 | -47.37 |
| passed_dR_lilj0point2             | 2574624.00 |  -0.00 |
| passed_Mll_4                      | 2574360.00 |  -0.01 |
| passed_lepisolation               | 2539242.00 |  -1.36 |
| passed_leptightID                 | 2531984.00 |  -0.29 |
| passed_lepclean                   |  514102.00 | -79.70 |
| passed_jetpt30                    |  514102.00 |   0.00 |
| passed_jeteta2opint4              |  354568.00 | -31.03 |
| passed_dijetpt100                 |  222616.00 | -37.21 |
| passed_nmergedjets_lepclean       |  154762.00 | -30.48 |
| passed_mergedjetsPt200            |  154762.00 |   0.00 |
| passed_mergedjetEta2opint4        |  153594.00 |  -0.75 |
| passed_ak4jetZm_40_180            |  137958.00 | -10.18 |
| passed_lepZmass40_180             |   81630.00 | -40.83 |
| passed_lepZ                       |   41242.00 | -49.48 |
| passed_mergedmass_40_180          |   36442.00 | -11.64 |
| passed_MergedolonlyHiggs          |    5728.00 | -84.28 |
| passed_particleNetZvsQCD0opint9   |    2620.00 | -54.26 |

**Comment by ram1123** [comment]
Thanks.
